proto/proto/ntx_builder.proto

```diff
@@ -0,0 +1,17 @@
+// Specification of the ntx-builder RPC.
+//
+// This provided access to the rollup data to the other nodes.
```

Comment:
Not sure if the comment on line 3 here is correct. Also, on line 1 we could probably write out "Network Transaction Builder RPC".